### ISSUE TITLE
selectCheckboxMenu: accessibility

### ISSUE THREAD

**Issue by adrianfiedler** [primefaces/primefaces#3019]
## 1) Environment
- PrimeFaces version: Primefaces 6.1
- Does it work on the newest released PrimeFaces version? No
- Does it work on the newest sources in GitHub? No
- Application server + version: JBoss EAP 7.0
- Affected browsers: Firefox ESR 52.5.0 (64-Bit)
- Screenreader: JAWS 18.0

## 2) Expected behavior
<p:selectCheckboxMenu> component should be read by screenreader similiar like <p:selectOneMenu> component. There it says "`label` dropdown, `selection`, enter text"

## 3) Actual behavior
<p:selectCheckboxMenu> component is read as "Read only input" by screenreaders.

## 4) Steps to reproduce
Read <p:selectCheckboxMenu> component with screenreader

**Comment by melloware** [comment]
If you could analyze p:selectOneMenu vs p:selectCheckboxMenu

https://github.com/primefaces/primefaces/blob/master/src/main/java/org/primefaces/component/selectonemenu/SelectOneMenuRenderer.java

https://github.com/primefaces/primefaces/blob/master/src/main/java/org/primefaces/component/selectcheckboxmenu/SelectCheckboxMenuRenderer.java

And see where the "role=" and any "aria=" are needed I can fix it.  I am definitely no expert in Accessibility but I know those two attributes are why control screenreaders.

**Comment by melloware** [comment]
@adrianfiedler I think it was as simple as this PR from everything I have read and comparing it to SelectOneMenu.  https://github.com/primefaces/primefaces/pull/4072
